clockify_client/models/project.py

```diff
@@ -21,17 +21,18 @@ def get_projects(self, workspace_id: str, params: dict | None = None) -> JsonTyp
         """
         if params:
             url_params = urlencode(params, doseq=True)
-            url = f"{self.base_url}/workspaces/{workspace_id}/projects?{url_params}"
+            path = f"/workspaces/{workspace_id}/projects?{url_params}"
         else:
-            url = f"{self.base_url}/workspaces/{workspace_id}/projects/"
+            path = f"/workspaces/{workspace_id}/projects/"
 
-        return self.get(url)
+        return self.get(path)
 
     def add_project(
         self,
         workspace_id: str,
         project_name: str,
         client_id: str,
+        *,
```

Comment:
Why are we accepting arbitrary arguments here?

Reply:
This confused me too the first time I saw it.  It forces callers to use kwargs for all the arugments after the `*`.  It is one way to solve the boolean trap pattern.

Reply:
Gooootcha! Okay.